cmd/insprd/operators/kafka/channels/client.go

```diff
@@ -22,7 +22,7 @@ func init() {
 
 // ChannelOperator is a client for channel operations on kafka
 type ChannelOperator struct {
-	k      *kafka.AdminClient
+	k      kafkaAdminClient
```

Comment:
@luiz-couto just curious. Why this change?

Reply:
I don't know haha I did not changed that. This diff was a bug when moving the PR from gitlab to github. This change is already in develop.

Reply:
I've looked and it actually was a change that @ptcar2009 did to allow the kafka admin client to be mocked. Now, the kafkaAdminClient in an interface.